tests/test_portal.py

```diff
@@ -264,27 +268,30 @@ def test_redirect_validation(self):
 
         client = self.add_client()
         client_url = client._redirect_uris
-        local_url = "http://{}/home?test".format(self.app.config.get('SERVER_NAME'))
+        local_url = "http://{}/home?test".format(
+            self.app.config.get('SERVER_NAME'))
         invalid_url = 'http://invalid.org'
 
         # validate redirect of /website-consent-script GET
         response = self.client.get('/website-consent-script/{}?redirect_url='
-                             '{}'.format(TEST_USER_ID, local_url))
+                                   '{}'.format(TEST_USER_ID, local_url))
```

Comment:
This would be more readable if you kept the string on the same line, eg:
```
response = self.client.get(
    '/website-consent-script/{}?redirect_url={}'.format(TEST_USER_ID, local_url)
)
```